### PR TITLE
Small fixes for Hub

### PR DESCRIPTION
**Pull request by misslivirose**
- Remove unused animation machine files
- Update plate pieces to get the right properties
- Add reset button script for walk override

### CODE REVIEW

File: DomainContent/Hub/Animation Shower/resetButton.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+// resetButton.js
+//
+// Author: Liv Erickson
+// Copyright High Fidelity 2018
+//
+// Licensed under the Apache 2.0 License
+// See accompanying license file or http://apache.org/
+//
+/* globals Audio, SoundCache */
+(function(){
```
Comment (thoys) [inline DomainContent/Hub/Animation Shower/resetButton.js]: missing space between () and {

File: DomainContent/Hub/Animation Shower/resetButton.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    var SOUND_URL = Script.resolvePath("Sounds/undo-notif.wav");
+
+    var walkRoles = ["walkFwd", "idleToWalkFwd", "walkBwdNormal"];
+    var sound = SoundCache.getSound(SOUND_URL);
+
+    function resetAnimations(){
```
Comment (thoys) [inline DomainContent/Hub/Animation Shower/resetButton.js]: missing space between () and {

File: DomainContent/Hub/Faucets/faucetServer.js [hunk 3]
```diff
@@ -27,6 +27,7 @@
             _this.entityID = entityID;
             sound = SoundCache.getSound(SOUND_URL);
             _this.on = Entities.getEntityProperties(_this.entityID, 'visible').visible;
+            Entities.editEntity(_this.entityID, {'alpha' : 1});
```
Comment (thoys) [inline DomainContent/Hub/Faucets/faucetServer.js]: Quotes not required,  remove space between key and colon

File: DomainContent/Hub/plates/ShatterPlatePiece.js [hunk 4]
```diff
@@ -81,7 +81,7 @@
         },
         collisionWithEntity : function(myID, theirID, collision) {
             if (canBreak) {
-                var properties = Entities.getEntityProperties(myID, 'velocity', 'position');
+                var properties = Entities.getEntityProperties(myID, ['velocity', 'position']);
```
Comment (thoys) [inline DomainContent/Hub/plates/ShatterPlatePiece.js]: good find

**Comment by misslivirose** [comment]
Intermediate PR for content in Spot - no testing needed

**Pull request merged**